src/components/burnerWallet/GuardianSetup.tsx

```diff
@@ -63,7 +63,7 @@ const GuardianSetup = () => {
   const [loading, setLoading] = useState(false);
 
   // 0 = 2 week default delay, don't do for demo
-  const [recoveryDelay, setRecoveryDelay] = useState(1);
+  const [recoveryDelay, setRecoveryDelay] = useState(0);
```

Comment:
Note for future changes, the production module has a minimum delay of 6 hours so this would need to be set to 6 hours under the hood. A small info message could be added when recovery is processed to tell the user to come back in 6 hours (kind of bad UX but the minimum delay is a useful security feature and it's needed as input to constructor, which impacts create2 address. So having consistent addresses means that we need this set to the same as the non-testnet version)